### PR TITLE
Allow writing rows to multiple sequences by externalId

### PR DESCRIPTION
**Pull request by hakontro**

### CODE REVIEW

File: src/main/scala/cognite/spark/v1/SequenceRowsRelation.scala [hunk 1]
```diff
@@ -187,10 +199,15 @@ class SequenceRowsRelation(config: RelationConfig, sequenceId: CogniteId)(val sq
...
-        .flatTap(_ => incMetrics(itemsCreated, rows.length))
+      import cats.instances.list._
+      projectedRows
+        .groupBy(_.externalId)
+        .toList
+        .traverse(
+          pair =>
+            client.sequenceRows
+              .insertByExternalId(pair._1.toString, columns, pair._2)
+              .flatTap(_ => incMetrics(itemsCreated, rows.length))) *> IO.unit
```
Comment (exyi) [inline src/main/scala/cognite/spark/v1/SequenceRowsRelation.scala:210]: Nitpick

```suggestion
        .traverse {
          case (externalId, rows) =>
            client.sequenceRows
              .insertByExternalId(externalId.toString, columns, rows)
              .flatTap(_ => incMetrics(itemsCreated, rows.length)) } *> IO.unit
```

**Review by exyi — APPROVED** [review]
I like this :)

Technically, we could cause someone trouble by adding a required field, but if there is not that many direct Spark users it's probably not a problem.

I think we should mention this somewhere in the docs, but I don't know what is the proper place - the current examples are so vague :sweat_smile:

File: src/main/scala/cognite/spark/v1/SequenceRowsRelation.scala (hunk 1 above)
Comment (hakontro) [reply src/main/scala/cognite/spark/v1/SequenceRowsRelation.scala:210]: Awesome, that's the one I was looking for!
Weirdly, `(externalId, rows)` typechecks, but then everything is in `externalId` and the `rows` is just junk I think

**Review by emelvarol-cognite — APPROVED** [review]
Awesome, I don't have anything to add besides id support but it is not a big deal.

**Pull request merged**